src/chainstate/coordinator/mod.rs

```diff
@@ -0,0 +1,307 @@
+use std::collections::VecDeque;
+use std::thread;
+use std::sync::{Arc, Mutex};
+use std::sync::mpsc::{self, Sender};
+use std::process;
+
+use burnchains::BurnchainHeaderHash;
+use chainstate::burn::{BlockHeaderHash, BlockSnapshot};
+use chainstate::stacks::{StacksBlock, TransactionPayload};
+use chainstate::stacks::db::StacksHeaderInfo;
+use chainstate::stacks::events::{StacksTransactionReceipt};
+
+#[derive(Debug, Clone)]
+struct PoxId;
+
+#[derive(Debug, Clone)]
+pub struct SortitionId (
```

Comment:
This is already defined as `SortitionId([u8; 32])` and is an opaque identifier used in the sortition DB.